README.md

```diff
@@ -943,7 +943,7 @@ Terser.minify(code, { mangle: { toplevel: true } }).code;
   - `"strict"` -- Advanced, all unquoted property names are mangled unless
     explicitly reserved.
 
-- `regex` (default: `null`) -— Pass a RegExp literal to only mangle property
+- `regex` (default: `null`) -— Pass a RegExp literal or [pattern](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html) string to only mangle property
```

Comment:
I wouldn't link to Java's documentation, that's only going to cause more confusion around Java != JavaScript. 😛 

```markdown
Pass a [RegExp literal or pattern string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp) to only mangle property matching the regular expression.
```

Reply:
Why not? I think that nobody will reach this section :D

I'll update link in PR few hours later